.github/workflows/release-keycloak-scim.yaml

```diff
@@ -0,0 +1,34 @@
+name: Build keycloak-scim and release
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Build with Gradle
+      run: gradle build --rerun-tasks clean
+
+    - name: Create Release with jar file
+      uses: "marvinpinto/action-automatic-releases@latest"
+      with:
+        repo_token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Is this a valid var name? I think it needs to be `GH_TOKEN`

Reply:
https://github.com/mitodl/ol-keycloak/blob/main/.github/workflows/release-ol-spi.yaml#L30